src/models/types.ts

```diff
@@ -0,0 +1,9 @@
+export interface RuleDefinition {
```

Comment:
nit: Do we have additional conflicts on 'Rule'? Can we drop 'Definition' on these?

Reply:
We can drop the definition part of the name. I'll do the same for the ResourceGraphQuery as well.